test/test_jit.py

```diff
@@ -6032,11 +6032,54 @@ def test_pow_int(x, y):
             # type: (float, int) -> float
             return math.pow(x, y)
 
+        def test_isnan(x):
+            # type: (float) -> bool
+            return math.isnan(x)
+
+        def test_asinh_int(x):
+            # type: (int) -> float
+            return math.asinh(x)
+
+        def test_asinh_float(x):
+            # type: (float) -> float
+            return math.asinh(x)
+
+        def test_atanh_int(x):
+            # type: (int) -> float
+            return math.atanh(x)
+
+        def test_atanh_float(x):
+            # type: (float) -> float
+            return math.atanh(x)
+
+        def test_cosh_float(x):
+            # type: (float) -> float
+            return math.cosh(x)
+
+        def test_cosh_int(x):
+            # type: (int) -> float
+            return math.cosh(x)
+
+        def test_sinh_int(x):
+            # type: (int) -> float
+            return math.sinh(x)
+
+        def test_sinh_float(x):
+            # type: (float) -> float
+            return math.sinh(x)
+
+        def test_tanh_int(x):
+            # type: (int) -> float
+            return math.tanh(x)
+
+        def test_tanh_float(x):
+            # type: (float) -> float
+            return math.tanh(x)
+
         self.checkScript(test_floor, (1.5,))
         self.checkScript(test_ceil, (1.5,))
         self.checkScript(test_log_int, (2,))
         self.checkScript(test_log_float, (2.0,))
-        self.checkScript(test_log_base_float, (2.0, 5.0))
```

Comment:
Why'd these get deleted?

Reply:
Not sure, I think it happend as part of the rebase, Ill look into adding these back in.

Reply:
Ive added that one in and will push, the other one self.checkScript(test_pow_float, (2.0, 2.0)) has a duplicate and that is why it was likely deleted